addons/mail/static/src/core/common/message_reaction_list.js

```diff
@@ -80,7 +80,7 @@ export class MessageReactionList extends Component {
     }
 
     hasSelfReacted(reaction) {
```

Comment:
Out of scope of the PR, but this one for example would be an extremely good candidate for compute, as the result is likely to never change, but the getter uses all `reaction.personas` so it will re-render for no reason on any of them changing.